tailwind.config.js

```diff
@@ -47,36 +60,89 @@ module.exports = {
         800: '#262626',
         900: '#171717'
       },
+      grayDark: {
+        0: '#000000',
+        50: '#171717',
+        100: '#262626',
+        200: '#404040',
+        300: '#525252',
+        400: '#737373',
+        500: '#A3A3A3',
+        600: '#D4D4D4',
+        900: '#FAFAFA'
+      },
       red: {
         50: '#FFF2F2',
         100: '#FFE6E6',
         200: '#FFBFBF',
         300: '#FF9999',
-        500: '#FF0000',
-        600: '#E60000'
+        500: '#ED1414',
+        600: '#CC0000',
+        700: '#A30000',
+      },
+      redDark: {
+        50: 'rgba(255, 72, 61, 0.15)',
+        100: 'rgba(255, 72, 61, 0.2)',
+        200: 'rgba(255, 72, 61, 0.3)',
+        300: '#CF3830',
+        500: '#FF483D',
+        600: '#FB6C64',
+        700: '#FFB5B0',
       },
       orange: {
         50: '#FFFAF4',
         100: '#FFF5EA',
         200: '#FFE5CA',
         300: '#FFD5A9',
         500: '#FF9629',
-        600: '#E68725'
+        600: '#E68725',
+        700: '#BF711F',
+      },
+      orangeDark: {
+        50: 'rgba(255, 159, 10, 0.15)',
+        100: 'rgba(255, 159, 10, 0.25)',
+        200: 'rgba(255, 159, 10, 0.4)',
+        300: '#CF3830',
+        500: '#FF9F0A',
+        600: '#FFB94C',
+        700: '#FFD99D'
       },
       green: {
         50: '#F2FBF4',
         100: '#E6F7E8',
         200: '#C0ECC6',
         300: '#9AE1A4',
-        500: '#02B31B',
-        600: '#02A118'
+        500: '#15E14E',
+        600: '#12BB41',
+        700: '#028614'
+      },
+      greenDark: {
+        50: 'rgba(33, 229, 41, 0.15)',
+        100: 'rgba(33, 229, 41, 0.25)',
+        200: 'rgba(33, 229, 41, 0.35)',
+        300: '#26A138',
+        500: '#21E529',
+        600: '#70E381',
+        700: '#ADEFB7'
       },
       blue: {
         50: '#F7FBFE',
         100: '#EFF6FD',
         200: '#D6EAF9',
+        300: '#BDDDF5',
         500: '#5AA9E6',
-      }
+        600: '#5198CF',
+        700: '#3A6C92'
+      },
+      blueDark: {
```

Comment:
let's not use camel case on CSS classes. darkblue or dark-blue is better.

Reply:
updated